app/javascript/components/enrollment/steps/Review.js

```diff
@@ -69,6 +69,7 @@ class Review extends React.Component {
               edit_mode={true}
               jurisdiction_path={this.props.jurisdiction_paths[this.props.currentState.patient.jurisdiction_id]}
               details={{ ...this.props.currentState.patient, blocked_sms: this.props.currentState.blocked_sms } || {}}
+              display_follow_up_flag={false}
```

Comment:
FYI - This flag is set so the follow-up flag is not displauyed on the monitoree review page at the end of the enrollment wizard.

Reply:
This thread should be resolved at this point. The original `edit_mode` flag is now used from the `Review` component to determine if the follow-up flag should/shouldn't be displayed on the Patient page.